crates/project-model/src/project_json.rs

```diff
@@ -319,9 +346,11 @@ struct CrateData {
     #[serde(default)]
     version: Option<semver::Version>,
     deps: Vec<Dep>,
+    // Cannot be a hashset because that would mean an inconsistent order.
+    #[serde(default)]
+    cfg_groups: Vec<String>,
```

Comment:
I think that if you say this is `FxHashSet`, order would be consistent across invocations.